File: java/Easy/DesignHashSet.java
package Easy;

class MyHashSet {

    boolean[] s;

    public MyHashSet() {
        s = new boolean[(int)1e6 + 1];
    }
    
    public void add(int key) {
        s[key] = true;
    }
    
    public void remove(int key) {
        s[key] = false;
    }
    
    public boolean contains(int key) {
        return s[key];
    }
}

File: python/Easy/DesignHashSet.py
class MyHashSet:

    def __init__(self):
        self.h = {}

    def add(self, key: int) -> None:
        self.h[key] = True

    def remove(self, key: int) -> None:
        if key in self.h:
            del self.h[key]

    def contains(self, key: int) -> bool:
        return key in self.h